unhandled exception in res.sendfile example

this code `res.status(err.status).end();` from [http://expressjs.com/en/api.html#res.sendFile](url) can throw unhandled exception when `ECONNABORTED ` occur and cause the worker died.
here logs from windows 10 x86

> [ERROR] nodeJS -[{ [Error: Request aborted] code: 'ECONNABORTED' }
> Error: Request aborted
>     at onaborted (...\node_modules\express\lib\response.js:973:15)
>     at Immediate._onImmediate (...\node_modules\express\lib\response.js:1015:9)
>     at processImmediate [as _immediateCallback] (timers.js:383:17)
> [ERROR] nodeJS -[GMT uncaughtException: Invalid status code: 0
> [ERROR] nodeJS -[RangeError: Invalid status code: 0
>     at ServerResponse.writeHead (_http_server.js:192:11)
>     at ServerResponse._implicitHeader (_http_server.js:157:8)
>     at ServerResponse.OutgoingMessage.end (_http_outgoing.js:573:10)
>     at ...\sources\server.js:128:36
>     at ...\node_modules\express\lib\response.js:413:22
>     at onaborted (...\node_modules\express\lib\response.js:975:5)
>     at Immediate._onImmediate (...\node_modules\express\lib\response.js:1015:9)
>     at processImmediate [as _immediateCallback] (timers.js:383:17)
> [WARN] nodeJS -[worker 3496 died.

--- a/_includes/api/en/4x/res-sendFile.md
+++ b/_includes/api/en/4x/res-sendFile.md
@@ -46,10 +46,8 @@ app.get('/file/:name', function (req, res, next) {
   var fileName = req.params.name;
   res.sendFile(fileName, options, function (err) {
     if (err) {
-      console.log(err);
-      res.status(err.status).end();
-    }
-    else {
+      next(err);
+    } else {
       console.log('Sent:', fileName);
     }
   });

